async/compiler/src/openxla/compiler/Dialect/Async/IR/test/verify.mlir

```diff
@@ -0,0 +1,21 @@
+// RUN: mlir-opt %s --iree-plugin=openxla-async -split-input-file -verify-diagnostics | FileCheck %s
```

Comment:
iree-opt, does it pass `ctest --test-dir build -R async`?

Reply:
Passed